lib/data/model/response_models/series.dart

```diff
@@ -0,0 +1,22 @@
+class Series {
+  final int? id;
+  final String? title;
+  final String? description;
+  final String? thumbnailPath;
+  final String? thumbnailExtension;
+
+  const Series(
+      {required this.id,
+      required this.title,
+      required this.description,
+      required this.thumbnailPath,
+      required this.thumbnailExtension});
+
+  String getThumbnailUrl(String size) {
```

Comment:
I do not see that you are using`size` argument - it has only one hardcoded value `portrait_medium` so we can hardcode it within the method.